404/model.html

```diff
@@ -0,0 +1,525 @@
+                </div>
+              </div>
+    
+    
+            </div>
+           
+        </div>
+        </div>
+        </nav>
+      </header>
+
+
+
+    <!-- Sara end -->
+
+    <!-- Qays -->
+    <div>
+        <div class="swiper-container">
+            <div class="swiper-slide ">
+                <img src="./imgs/female_model/1.jpg" alt="female_model_1">
+                <img src="./imgs/female_model/2.jpg" alt="female_model_2">
+                <img src="./imgs/female_model/3.jpg" alt="female_model_3">
+                <img src="./imgs/female_model/4.jpg" alt="female_model_4">
+                <img src="./imgs/female_model/5.jpg" alt="female_model_5">
+                <img src="./imgs/female_model/6.jpg" alt="female_model_6">
+            </div>
+        </div>
+    </div>
+    <div class="container mb-5">
+        <div class="row">
+            <div class="col-lg-8 col-md-12 ">
+                <div>
+                    <a class="btn btn-light btn-lg rounded-circle mr-3 mb-3 mt-5 ">
+                        <i class="fas fa-arrow-left"></i>
+                    </a>
+                    <a class="btn btn-light btn-lg rounded-circle mr-3 mb-3 mt-5">
+                        <i class="fas fa-arrow-right"></i>
+                    </a>
+                </div>
+                <h1 class="portfolio-title">Female models</h1>
+                <div style="max-width: 600px;">
+
+                    <p><strong> Drops of rain could be heard hitting the pane. </strong></p>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin dapibus porttitor eros vitae
+                        molestie,
+                        consectetur urna id, dapibus nulla. Suspendisse potenti. Proin ullamcorper nulla vel est
+                        condimentum
+                        bibendum. Phasellus sollicitudin.</p>
+                    <ul class="ml-5">
+                        <li>Brainstorming</li>
+                        <li>Designing</li>
+                        <li>Building</li>
+                        <li>Presenting</li>
+                    </ul>
+                    <div class="accordion mr-auto" id="accordionExample">
+                        <div class="card">
+                            <div class="card-header" id="headingOne">
+                                <h2 class="mb-0">
+                                    <button class="btn btn-link" type="button" data-toggle="collapse"
+                                        data-target="#collapseOne" aria-expanded="false" aria-controls="collapseOne">
+                                        Materials
+                                    </button>
+                                </h2>
+                            </div>
+
+                            <div id="collapseOne" class="collapse" aria-labelledby="headingOne"
+                                data-parent="#accordionExample">
+                                <div class="card-body">
```

Comment:
Well done on achieving these collapsable containers. It looks super smooth with the opening and closing animation! 👍